krux_redis/cli.py

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8 -*-
+#
+# © 2017 Krux, A Salesforce Company
+#
+
+from __future__ import absolute_import
+
+from krux.cli import get_group
+
+
+DEFAULT_TIMEOUT = 0.5   # in seconds
+
+
+# Designed to be called from krux.cli, or programs inheriting from it
+def add_redis_cli_arguments(parser):
```

Comment:
I'm going to say that's out of scope for my current work, which is focused mainly around getting the structure in place to do my Sentinel work.